stetho/src/main/java/com/facebook/stetho/inspector/elements/ObjectDescriptor.java

```diff
@@ -53,7 +53,18 @@ public void setAttributesAsText(Object element, String text) {
   }
 
   @Override
-  public void getStyles(Object element, StyleAccumulator accumulator) {
+  public void getStyleRules(Object element, StyleRuleAccumulator accumulator) {
+
```

Comment:
nit: to match the style of the rest of the code, empty methods should have exactly 1 newline after the opening brace